learn/work/calibration-splits/index.qmd

```diff
@@ -0,0 +1,478 @@
+---
+title: "Automatic calibration splits for resamples"
+categories:
+  - post-processing
+  - calibration
+  - rsample
+type: learn-subsection
+weight: 6
+description: | 
+  Learn how tidymodels generates automatic calibration sets for post-processing.
+toc: true
+toc-depth: 2
+include-after-body: ../../../resources.html
+---
+
+```{r}
+#| label: "setup"
+#| include: false
+#| message: false
+#| warning: false
+source(here::here("common.R"))
+```
+
+```{r}
+#| label: "load"
+#| include: false
+#| message: false
+#| warning: false
+library(tidymodels)
+
+pkgs <- c("tidymodels", "rsample")
+
+theme_set(theme_bw() + theme(legend.position = "top"))
+```
+
+While preprocessing is the transformation of the predictors prior to a model fit, post-processing is the transformation of the predictions after the model fit. This could be as straightforward as limiting predictions to a certain range of values to as complicated as transforming them based on a separate calibration model. 
+
```

Comment:
Below we are using the term **primary model** which [we just started using](https://github.com/search?q=org:tidymodels++%22primary+model%22&type=code). I like the term but i think it be nice to properly define it, in terms of pre/model/post diagram/terminology